src/ApiConnectors/GeneralLedgerAccountsConnector.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace PhpTwinfield\ApiConnectors;
+
+use PhpTwinfield\Services\FinderService;
+use PhpTwinfield\GeneralLedgerAccount;
+
+/**
+ * A facade to make interaction with the the Twinfield service easier when trying to retrieve or send information about
+ * General Ledger Accounts.
+ */
+class GeneralLedgerAccountsConnector extends BaseApiConnector
+{
+    /**
+     * List all General Ledger Accounts.
+     *
+     * @param string $pattern  The search pattern. May contain wildcards * and ?
+     * @param int    $field    The search field determines which field or fields will be searched. The available fields
+     *                         depends on the finder type. Passing a value outside the specified values will cause an
+     *                         error.
+     * @param int    $firstRow First row to return, useful for paging
+     * @param int    $maxRows  Maximum number of rows to return, useful for paging
+     * @param array  $options  The Finder options. Passing an unsupported name or value causes an error. It's possible
+     *                         to add multiple options. An option name may be used once, specifying an option multiple
+     *                         times will cause an error.
+     *
+     * @return array The General Ledger Accounts found.
+     */
+    public function listAll(
+        string $pattern = '*',
+        int $field = 0,
+        int $firstRow = 1,
+        int $maxRows = 100,
+        array $options = []
+    ): array {
+        $response = $this->getFinderService()->searchFinder(
+            FinderService::TYPE_DIMENSIONS_PROJECTS,
```

Comment:
If this connector is intended for general ledger accounts, then `TYPE_DIMENSIONS_FINANCIALS` would be more appropriate.

The results should also be limited to level 1 dimensions only.

Reply:
True about the dimension. Was lazy as they both reference to DIM. Will change that. Can you elaborate the dimension limit wish as I am not sure what you intend.

After I will submit a clean, one commit, PR for this change.